ksp_plugin/interface_flight_plan.cpp

```diff
@@ -521,13 +521,21 @@ void __cdecl principia__FlightPlanRenderedApsides(
       GetFlightPlan(*plugin, vessel_guid).GetAllSegments();
   DiscreteTrajectory<World> rendered_apoapsides;
   DiscreteTrajectory<World> rendered_periapsides;
-  plugin->ComputeAndRenderApsides(celestial_index,
-                                  flight_plan,
-                                  flight_plan.begin(), flight_plan.end(),
-                                  FromXYZ<Position<World>>(sun_world_position),
-                                  max_points,
-                                  rendered_apoapsides,
-                                  rendered_periapsides);
+  for (auto const& segment : flight_plan.segments()) {
+    DiscreteTrajectory<World> segment_rendered_apoapsides;
+    DiscreteTrajectory<World> segment_rendered_periapsides;
+    plugin->ComputeAndRenderApsides(
+        celestial_index,
+        flight_plan,
+        segment.begin(),
```

Comment:
Let's put `begin` and `end` on the same line, as it was originally formatted.  (Same in the other places.)